cmd/osv-scanner/main_test.go

```diff
@@ -349,59 +343,95 @@ func TestRun(t *testing.T) {
 			args:         []string{"", "--format", "sarif", "--config", "./fixtures/osv-scanner-empty-config.toml", "./fixtures/locks-many/package-lock.json"},
 			wantExitCode: 1,
 			wantStdout: `
-				{
-					"version": "2.1.0",
-					"$schema": "https://json.schemastore.org/sarif-2.1.0.json",
-					"runs": [
-						{
-							"tool": {
-								"driver": {
-									"informationUri": "https://github.com/google/osv-scanner",
-									"name": "osv-scanner",
-									"rules": [
-										{
-											"id": "vulnerable-packages",
-											"shortDescription": {
-												"text": "This manifest file contains one or more vulnerable packages."
-											}
-										}
-									]
-								}
-							},
-							"artifacts": [
-								{
-									"location": {
-										"uri": "fixtures/locks-many/package-lock.json"
-									},
-									"length": -1
-								}
-							],
-							"results": [
-								{
-									"ruleId": "vulnerable-packages",
-									"ruleIndex": 0,
-									"level": "warning",
-									"message": {
-										"text": "+-----------+-------------------------------------+------+-----------------+---------------+\n| PACKAGE \u0026nbsp; | VULNERABILITY ID \u0026nbsp; \u0026nbsp; \u0026nbsp; \u0026nbsp; \u0026nbsp; \u0026nbsp; \u0026nbsp; \u0026nbsp; \u0026nbsp; \u0026nbsp;| CVSS | CURRENT VERSION | FIXED VERSION |\n+-----------+-------------------------------------+------+-----------------+---------------+\n| ansi-html | https://osv.dev/GHSA-whgm-jr23-g3j9 | 7.5 \u0026nbsp;| 0.0.1 \u0026nbsp; \u0026nbsp; \u0026nbsp; \u0026nbsp; \u0026nbsp; | 0.0.8 \u0026nbsp; \u0026nbsp; \u0026nbsp; \u0026nbsp; |\n+-----------+-------------------------------------+------+-----------------+---------------+"
-									},
-									"locations": [
-										{
-											"physicalLocation": {
-												"artifactLocation": {
-													"uri": "fixtures/locks-many/package-lock.json"
-												}
-											}
-										}
-									]
-								}
-							]
-						}
-					]
-				}
+            {
+              "version": "2.1.0",
+              "$schema": "https://json.schemastore.org/sarif-2.1.0.json",
+              "runs": [
+                {
+                  "tool": {
+                    "driver": {
+                      "informationUri": "https://github.com/google/osv-scanner",
+                      "name": "osv-scanner",
+                      "rules": [
+                        {
+                          "id": "GHSA-whgm-jr23-g3j9",
```

Comment:
Should we have an end-to-end test here that includes CVE, Ecosystem ID examples etc?

Reply:
Added a bigger test with CVE's and multiple ecosystems, not as an end-to-end test but a unit test since it makes the path resolution easier. It already found some problems with CVEs not actually being the display ID, which should be fixed now. 

Do you still want a bigger end-to-end test as well?